salt/modules/napalm_mod.py

```diff
@@ -742,37 +742,6 @@ def netmiko_config(*config_commands, **kwargs):
     return __salt__["netmiko.send_config"](config_commands=config_commands, **kwargs)
 
 
-@proxy_napalm_wrap
-def netmiko_conn(**kwargs):
-    """
-    .. versionadded:: 2019.2.0
-
-    Return the connection object with the network device, over Netmiko, passing
-    the authentication details from the existing NAPALM connection.
-
-    .. warning::
-
-        This function is not suitable for CLI usage, more rather to be used
-        in various Salt modules.
-
-    USAGE Example:
-
-    .. code-block:: python
-
-        conn = __salt__['napalm.netmiko_conn']()
-        res = conn.send_command('show interfaces')
-        conn.disconnect()
-    """
-    salt.utils.versions.warn_until(
-        3007,
-        "This 'napalm_mod.netmiko_conn' function as been deprecated and "
-        "will be removed in the {version} release, as such, it has been "
-        "made an internal function since it is not suitable for CLI usage",
-    )
-    return _netmiko_conn(**kwargs)
-
-
-@proxy_napalm_wrap
```

Comment:
I don't think we want to these decorator from the `junos_rpc` function which we aren't removing.

Reply:
yea